tools/chafa/chicle-coregraphics-loader.c

```diff
@@ -0,0 +1,445 @@
+/* -*- Mode: C; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- */
+
+/* Copyright (C) 2025 Hans Petter Jansson
+ *
+ * This file is part of Chafa, a program that shows pictures on text terminals.
+ *
+ * Chafa is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Chafa is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Chafa.  If not, see <http://www.gnu.org/licenses/>. */
+
+#include "config.h"
+
+#ifdef HAVE_COREGRAPHICS
```

Comment:
Since the module is compiled and linked conditionally, there is no need to surround it with `#ifdef`.